test/utils.test.ts

```diff
@@ -22,4 +22,25 @@ describe('utils', () => {
             expect(compareText('qwe_AsD_zxc', 'asd', { ignoreCase: true, containing: true }).result).toBe(true)
         })
     })
+    describe('compareTextWithArray', () => {
```

Comment:
Can you please add one `.not` test

Reply:
Done

Reply:
I'm sorry for confusing, but I mean something like this
```
compareTextWithArray(' FOO ', ['foO', 'bar'], { trim: true, ignoreCase: true, isNot: true })
```

Reply:
see 
https://github.com/webdriverio/expect-webdriverio/blob/24c23ec500b07e3fd088faa375104bdbfc73958d/test/matchers/element/toBeDisabled.test.ts#L74
and
https://github.com/webdriverio/expect-webdriverio/blob/24c23ec500b07e3fd088faa375104bdbfc73958d/test/matchers/element/toBeDisabled.test.ts#L86

Reply:
It's impossible to add `isNot` as argument because only these 3 elements are declared `export const compareTextWithArray = (actual: string, expectedArray: Array<string>, { ignoreCase = false, trim = false, containing = false}) => {`
If not, should I create a new test file `toHaveText.test.ts  in ` expect-webdriverio/test/matchers/element/

Reply:
yeah, you are right! It would be really helpful if you can!

Reply:
Done, now the coverage of toHaveText.ts is 100%